apps/dailyscrum/src/app/orgs/[orgId]/dashboard/queries.ts

```diff
@@ -0,0 +1,82 @@
+import { getOrgByHashId } from "@/lib/services";
+import { createClient } from "@/lib/supabase/client";
+import { Database } from "@/lib/supabase/database";
+import { unstable_cache } from "next/cache";
+
+/**
+ * Retrieves and lists daily scrum update form ids for a specific organization.
+ *
+ * @param {number} orgId - The ID of the organization to retrieve form ids for.
+ */
+async function listDailyScrumUpdateFormIdsOfOrg(orgId: number) {
+  const client = createClient<Database>();
+
+  return unstable_cache(
+    async () => {
+      return client
+        .from("daily_scrum_update_forms")
+        .select("id")
+        .eq("org_id", orgId);
+    },
+    [`daily-scrum-update-form-ids-of-org-${orgId}`],
+    {
+      tags: [`daily-scrum-update-form-ids-of-org-${orgId}`],
+      revalidate: false,
+    }
+  )();
+}
+
+/**
+ * Retrieves and lists daily scrum update entries based on the organization hash ID and date.
+ *
+ * @param {string} orgHashId - The hash ID of the organization
+ * @param {string} date - The ISO formatted date string. e.g. 2024-03-17
+ */
+export async function listDailyScrumUpdateEntries(
+  orgHashId: string,
+  date: string
+) {
```

Comment:
A function that retrieves data from the `daily_scrum_update_entries` table. It filters by organization and date.

Reply:
[Next comment](https://github.com/kiwiy-is/dailyscrum/pull/36/files#r1529910781)